apps/frontend/src/components/BasicLayout.module.scss

```
@@ -1,5 +1,6 @@
 .vod-layout {
-    font-family: 'Righteous', 'Poppins', 'Microsoft YaHei', sans-serif;
+    font-family: 'Geist', 'Inter', 'Microsoft YaHei', sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)